janus.c

```diff
@@ -3360,6 +3361,25 @@ gboolean janus_transport_is_auth_token_valid(janus_transport *plugin, const char
 gboolean carbyne_janus_transport_is_sanityhealthcheck_token_valid(janus_transport *plugin, const char *token) {
     return token && carbyne_janus_auth_sanityhealthcheck_signature(token);
 }
+long* getLong(char *str)
```

Comment:
Why not use `strtol`  or any other existing functions that do the same?

Reply:
getlong extracts the number from the line ignoring the letters by using both isdigit and strtol